ni_measurementlink_service/measurement/info.py

```diff
@@ -56,6 +56,10 @@ class ServiceInfo(NamedTuple):
         - Key: "ni/service.tags"
            - Expected format: serialized JSON string of an array of strings
            - Example: "[\"powerup\", \"current\"]"
+    - Programminglanguage
```

Comment:
I'm wondering if we should even document this annotation. We don't really want the user changing this or using it, do we?

Reply:
True, will remove it.